GetStream/stream-chat-swift
# Message Action title now supports 2 lines of text instead of 1

### 🎯 Goal

Do not truncate the title of message action in message actions pop up.

### 🛠 Implementation

* `titleLabel` now supports showing 2 lines of text instead of 1
* fixes indentation of the pop up

### 🎨 Showcase

| Before | After |
| ------ | ----- |
| ![Simulator Screen Shot - iPod touch (7th generation) - 2022-06-10 at 12 48 58](https://user-images.githubusercontent.com/4924709/173050846-b4537e12-26d4-444c-bfd4-a77332f15e86.png) |  ![Simulator Screen Shot - iPod touch (7th generation) - 2022-06-10 at 12 47 57](https://user-images.githubusercontent.com/4924709/173050877-360306b1-ced7-4eb0-aa83-8193f2be5ff4.png) |

### 🧪 Manual Testing Notes

1. Use title that takes 2lines of text for one of the support message actions
2. Run the app.
3. Invoke the message action pop up and verify it works fine.

### ☑️ Contributor Checklist

### 🎁 Meme

![_Provide a funny gif or image that relates to your work on this pull request. (Optional)_](https://media.giphy.com/media/l1J9qemh1La8b0Rag/giphy.gif)
